metricbeat/beater/runner.go

```diff
@@ -0,0 +1,59 @@
+package beater
+
+import (
+	"sync"
+
+	"github.com/elastic/beats/libbeat/publisher"
+)
+
+// ModuleRunner is a facade for a ModuleWrapper that provides a simple interface
```

Comment:
Now that it has grown to be usable outside of Metricbeat's `Beater` implementation, I agree.